templates/spec/system/checkout_spec.rb

```diff
@@ -364,7 +364,7 @@
       click_on "Place Order"
 
       order = Spree::Order.last
-      expect(page).to have_current_path(spree.token_order_path(order, order.guest_token))
+      expect(page).to have_current_path(spree.order_path(order))
```

Comment:
From the commit message:

Expected behavior for the specs
----------------------------------

Given I am signed in
And I am checking out an order
When I finalize the order (click on "Place Order")
Then I should be redirected to the order path

Actual behavior of the broken specs
-----------------------------------

I'm redirected to the guest order path.

Cause
-----

`Spree::CheckoutController#completion_route` uses `spree_current_user`
to determine the order completion route, which is the order path if
there is a current user, and the guest order path if there is none.
However, the specs were previously stubbing only
`try_spree_current_user` and not `spree_current_user`. As a result, the
specs was redirecting incorrectly to the guest order path.

Solution
--------

The spec bugs were revealed in the previous commit, where all
`try_spree_current_user` references were changed to
`spree_current_user`. This commit fixes the expected order paths for the
specs.